gluten-ut/src/test/scala/io/glutenproject/utils/velox/VeloxNotSupport.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.glutenproject.utils.velox
+
+import io.glutenproject.utils.NotSupport
+import org.apache.spark.sql.catalyst.expressions._
+
+object VeloxNotSupport extends NotSupport {
+  override lazy val notSupportSuiteLList: Map[String, Map[String, ExpressionInfo]] = Map(
+    (classOf[NullExpressionsSuite].getSimpleName, Map.empty)
```

Comment:
velox has support NullExpressionsSuite, and merged

Reply:
The value of the key 'NullExpressionSuite' is `Map.empty`, so it will returen Seq.empty, and then `NullExpressionSuite` will be tested for Velox.  But this's actually confused, `NullExpressionsSuite` is supported but in the `notSupportSuiteList`.

Reply:
let's add a `supportedSuiteLists` to avoid confusing